src/fundus/publishers/de/hessenschau.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime
+from typing import Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_date_parsing,
+)
+
+
+class HessenschauParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = CSSSelector("p.copytext__text.text__copytext")
+        _subheadline_selector = CSSSelector("h2.text__headline.copytext__headline")
+
+        @attribute
+        def body(self) -> ArticleBody:
+            return extract_article_body_with_selector(
+                self.precomputed.doc,
+                paragraph_selector=self._paragraph_selector,
+                subheadline_selector=self._subheadline_selector,
+            )
+
+        @attribute
+        def title(self) -> Optional[str]:
+            return self.precomputed.meta.get("og:title")
+
+        @attribute
+        def publishing_date(self) -> Optional[datetime]:
+            return generic_date_parsing(self.precomputed.meta.get("article:published_time"))
```

Comment:
Could you add `authors` and `topics` as well? You can find both of them in the meta tags. Just refer to other parsers to see how to extract them.